specification/liftrmongodb/MongoDB.Atlas.Management/main.tsp

```diff
@@ -58,6 +58,34 @@ model UserDetails {
   companyName?: string;
 }
 
+@doc("Base patch resource properties that can be extended for arm resources.")
+model BaseResourcePatchProperties {
+  @doc("Marketplace details of the resource.")
+  @visibility(Lifecycle.Create, Lifecycle.Read)
+  marketplace?: MarketplaceDetails;
```

Comment:
Why you have a PatchProperties, and its property is Create+Read (no Update)?

Reply:
this is copied from https://github.com/Azure/azure-rest-api-specs/pull/35465/files#diff-f9252771918abdd5ff741d0976288e6f4ce3d21bbb445f77ff63b419eb2897bcR90-R101
```
@doc("Base resource properties that can be extended for arm resources.")
model BaseResourceProperties {
  @doc("Marketplace details of the resource.")
  @visibility(Lifecycle.Create, Lifecycle.Read)
  marketplace: MarketplaceDetails;

  @doc("Details of the user.")
  user: UserDetails;

  @doc("Provisioning state of the resource.")
  @visibility(Lifecycle.Read)
  provisioningState?: ResourceProvisioningState;
}
```
and update all the properties as optional

Reply:
I am asking why a property that exclude visibility=Update appears in PATCH only model.

Reply:
Yes, I think this makes sense, we could simply remove this property for Update method.

Reply:
We would need to still have it + Read, as it would be in response of PATCH.